src/test/util/setup_common.cpp

```diff
@@ -88,6 +89,15 @@ std::ostream& operator<<(std::ostream& os, const uint256& num)
     return os;
 }
 
+struct NetworkSetup
```

Comment:
This could probably use at least one line of doc explaining it's existence? Given it's only here to facilitate Windows sockets initialization, and doesn't do anything for non-windows platforms.

Reply:
> the code difference between platforms looks rather like implementation details.

The only code in `SetupNetworking` is Windows code to call `WSAStartup()`?

Reply:
That's correct.